lookout/style/format/descriptions.py

```diff
@@ -184,3 +185,48 @@ def describe_rule_parts_ordinal(feature: OrdinalFeature, name: str,
     """
     cmp, threshold, _ = splits[0]
     return _format_int(cmp, threshold, name)
+
+
+def get_error_description(vnode: VirtualNode, y_pred: int) -> str:
+    """
+    Return the comment with regard to the correct node class.
+
+    :param vnode: Node to describe.
+    :param y_pred: The index of the correct node class.
+    :return: String comment.
+    """
+    column = vnode.start.col
+    if y_pred == CLASS_INDEX[CLS_NOOP]:
+        return "%s at column %d should be removed." % (CLASS_REPRESENTATIONS[vnode.y], column)
+    elif vnode.y == CLASS_INDEX[CLS_NOOP]:
+        return "%s should be inserted at column %d." % (CLASS_REPRESENTATIONS[y_pred], column)
+    else:
+        return "Replace %s with %s at column %d." % (
+            CLASS_REPRESENTATIONS[vnode.y], CLASS_REPRESENTATIONS[y_pred], column)
+
+
+def get_code_chunk(lang: str, code_lines: Sequence[str], line_number: int) -> str:
+    """
+    Return nice code snippet that can be inserted to github message.
+
+    :param lang: Code language.
+    :param code_lines: Code lines.
+    :param line_number: 1-based line number to print
+    :return: Code snippet.
+    """
+    lines = list(range(max(0, line_number - 2), line_number + 1))
+    code_chunk = "\n".join("%d|%s" % (l, code_lines[l]) for l in lines)
+    return "```%s\n%s\n```\n" % (lang, code_chunk)
+
+
+def rule_to_comment(rule: Rule, number: int, feature_extractor: FeatureExtractor) -> str:
+    """
+    Return the comment for the rule.
+
+    :param rule: The rule to convert to string.
+    :param number: Rule number.
```

Comment:
What is "rule number"? Is it the count of the rules? Is it the index? If it's an index, then where?